xmatters.json

```diff
@@ -2318,6 +2318,722 @@
                 }
             ],
             "versions": "EQ(*)"
+        },
+        {
+            "action": "list groups",
+            "description": "Get information about multiple groups matching a property name/value",
```

Comment:
The description seems incorrect based on inputs params. There is no property name/value input params

Reply:
Modified the description.